src/pages/Run/RunPage.tsx

```diff
@@ -45,6 +48,8 @@ type RunPageProps = {
 
 const RunPage: React.FC<RunPageProps> = ({ run, params }) => {
   const { t } = useTranslation();
+  const plContext = useContext(PluginsContext);
+  const plugins = plContext.getPluginsBySlot('run-tab');
```

Comment:
I would like to see stricter constraints on strings like `run-tab`. Either have that string as a constant somewhere, or have `getPluginsBySlot` have more type safety and only accept valid slot names.

Reply:
Good point. Added strict type for getPluginBySlot